packages/components/tree-select/__tests__/tree-select.test.tsx

```diff
@@ -45,13 +45,12 @@ const createComponent = ({
       }
     },
     render() {
-      return h(
-        TreeSelect,
-        {
-          ...this.$data,
-          ref: (val: object) => (wrapperRef.value = val),
-        },
-        slots
+      return (
+        <TreeSelect
+          {...this.$data}
```

Comment:
Prefer using ref instead of data property and don't mix data and props.

FYI.
```ts
  const data = ref([
    {
      value: 1,
      label: '一级 1',
      children: [
        {
          value: 11,
          label: '二级 1-1',
          children: [
            {
              value: 111,
              label: '三级 1-1',
            },
          ],
        },
      ],
    },
  ])

  const wrapper = mount({
    render() {
      return (
        <TreeSelect
          data={data.value}
          renderAfterExpand={false}
          {...props}
          modelValue={value.value}
          onUpdate:modelValue={(val: string) => (value.value = val)}
          ref={(val: object) => (wrapperRef.value = val)}
          v-slots={slots}
        />
      )
    },
  })
```